front/components/sparkle/AppContentLayout.tsx

```diff
@@ -96,44 +87,12 @@ export default function AppContentLayout({
       />
       <div
         className={cn(
-          "relative h-full w-full flex-1 flex-col overflow-x-hidden overflow-y-hidden",
+          "relative h-full w-full flex-1 overflow-hidden",
```

Comment:
I feel like we can remove this `h-full` too but maybe we should leave it to a remove-all-the-useless-h-full mission

Reply:
probably overflow-hidden too can be removed